presto-native-execution/presto_cpp/main/http/tests/HttpTest.cpp

```diff
@@ -35,6 +35,9 @@ int main(int argc, char** argv) {
 }
 
 namespace {
+
+enum Protocol : int8_t { http = 0, https = 1 };
```

Comment:
We could use a boolean for this?

Reply:
Initially I was thinking to use boolean, later moved with enum just for readability. Let me move back to boolean, I think that's clear as well.